OxfordBodleian/Ullendorff/BDLeu6.xml

```diff
@@ -130,21 +173,96 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="right">11</dim>
                                         <dim type="left">8</dim>
                                     </dimensions>
+                                   <note>The characters per line are 12–18.</note>
+                                    <ab type="punctuation" subtype="Dividers">One black lines separates the stanzas on <locus target="#7r"/>.</ab>
+                                    <ab type="pricking">The quire is unruled.</ab>
                                 </layout>
                             </layoutDesc>
-                            
                         </objectDesc>
                         
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                                <locus from="9ra" to="68vb"/>
+                                <date notBefore="1700" to="1800"/>
+                              <desc>Fine and regular handwriting. The ligature <foreign xml:lang="gez">ግዚ</foreign> is used e.g. on <locus target="#1r"/>.
+                                    </desc>
+                           <seg type="ink">Black, red.</seg>
+                                <seg type="rubrication">
+                                    Few lines on the incipit page of <ref target="#ms_i1">the first text</ref>, alternating with black lines; 
+                                    the incipits of the sections of <ref target="#ms_i1">the first text</ref>; the first word of <ref target="#ms_i2">the second text</ref>;
+                                    holy names, included that of the saint; elements of the punctuation, including text dividers, and numerals.
+                                </seg>
+                          
                             </handNote>
+                            <handNote xml:id="h2" script="Ethiopic">
```

Comment:
Here, it would be useful to add the attribute corresp="#a1" (or add a reference to h2 in a1). Maybe this relation could also be made explicit in the printed text ("hand of addition 1:....") or something similar. This for sure also comes up in other manuscripts, and we might discuss it at another point.